data/random_genes.tsv

```diff
@@ -0,0 +1,108 @@
+gene	classification
+HECTD4	neither
```

Comment:
To what extent are the genes whose mutations are predictable in random_genes/top_genes just the genes that are in the Vogelstein set?

Reply:
Yeah, good point - by eye it looks like a lot of them are, but I'll add some code to the notebook to count the overlap.

Reply:
Here's the overlap between all genes in the gene sets:

![image](https://user-images.githubusercontent.com/2345877/115564608-9a1f6c00-a286-11eb-874b-143667f0ecc1.png)

And between the significantly predictable genes:

![image](https://user-images.githubusercontent.com/2345877/115564635-a4da0100-a286-11eb-8cbb-bef30040a4c6.png)

I haven't done any quantitative overlap analysis, but looking at the second diagram 12/43 significant genes in the most mutated gene set are also in Vogelstein, and 7/10 significant genes in the random gene set are also in Vogelstein. So for the random genes, most of the signal is also captured by the Vogelstein gene set, but maybe not so much for the most mutated genes (although some of it is).